forge-gui/res/cardsfolder/upcoming/effluence_devourer.txt

```diff
@@ -0,0 +1,12 @@
+Name:Effluence Devourer
+ManaCost:B R G
+Types:Creature Crocodile
+PT:4/3
+K:Blitz:B R G
+T:Mode$ Sacrificed | ValidCard$ Creature | ValidPlayer$ You | Execute$ TrigEffect | TriggerZones$ Battlefield | TriggerDescription$ Whenever you sacrifice CARDNAME or another creature, it perpetually gains “{2}, Exile this card from your graveyard: Create an X/X green Ooze creature token, where X is this card's power. Activate only as a sorcery.”
+SVar:TrigEffect:DB$ Effect | ValidCard$ Creature | EffectZone$ Command | StaticAbilities$ PerpetualAbility | Name$ Effluence Devourer's Perpetual Effect | Duration$ Permanent
+SVar:PerpetualAbility:Mode$ Continuous |  Affected$ Creature | AddAbility$ ABToken | Defined$ Remembered | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ {2} Exile this card from your graveyard: Create an X/X green Ooze creature token, where X is this card's power. Activate only as a sorcery.
+SVar:ABToken:AB$ ChangeZone | Cost$ 2 ExileFromGrave<1/CARDNAME> | ActivationZone$ Graveyard | Origin$ All | Destination$ Exile | EffectZone$ Command | SorcerySpeed$ True | SubAbility$ DBToken | StackDescription$ Exile this card from your graveyard | SpellDescription$ Create an X/X green Ooze creature token, where X is this card's power.
+SVar:DBToken:DB$ Token | TokenScript$ g_x_x_ooze | TokenPower$ X | TokenToughness$ X | TokenOwner$ You | TokenAmount$ 1 | StackDescription$ Create an X/X green Ooze creature token, where X is this card's power.
```

Comment:
There should not be a ChangeZone line... the exile is part of the cost

Line 9 should be a Token line

Please note `TokenOwner$ You` and `TokenAmount$ 1` are defaults so you don't need them